spec/shopify_transporter/exporters/magento/soap_spec.rb

```diff
@@ -56,14 +56,22 @@ def stub_login_call(mock_client, body: { login_response: { login_return: '123' }
             expect { Soap.new(init_params).call(:test_call, {}) }.to raise_error(Soap::FailedLoginError, expected_error_message)
           end
 
-          it 'creates a session correctly if login response contains a session id' do
+          it 'creates a session correctly if the login response contains a session id in format A' do
             mock_client = spy('mock_client')
             stub_client_call(mock_client)
             stub_login_call(mock_client)
 
             Soap.new(init_params).call(:test_call, {})
           end
 
+          it 'creates a session correctly if login response contains a session id in format B' do
```

Comment:
minor: prefer being more explicit: `with keys : login_response_param and : result `